src/main/java/dk/creditoro/client/view/frontpage/FrontpageController.java

```diff
@@ -130,4 +130,9 @@ public void drawerAction(ActionEvent actionEvent) {
             closeBtn.play();
         }
     }
+
+    @FXML
+    public void btnSearch(ActionEvent actionEvent) {
+        viewHandler.openView(Views.FRONTPAGE);
```

Comment:
Instead of changing to the same view, we could set focus to search bar
```suggestion
        searchTextField.requestFocus();
```